python/ray/autoscaler/autoscaler.py

```diff
@@ -246,6 +307,19 @@ def __init__(self,
         self.last_update_time = 0.0
         self.update_interval_s = update_interval_s
 
+        # Node launchers
+        self.launch_queue = queue.Queue()
+        self.num_launches_pending = Counter()
+        max_batches = max_concurrent_launches // max_launch_batch
+        max_batches += (max_concurrent_launches % max_launch_batch > 0)
```

Comment:
This is kind of a confusing expression, if we just want to make sure max_batches is > 0, could we do `max_batches = max(1, max_concurrent_launches // max_launch_batch)`?

Reply:
I agree this is ugly. It's computing the integer ceiling, to ensure that we will have sufficient launchers to hit he ```max_concurrent_launches``` limit. 

Options are:
  * Your suggestion which I think has less-desirable semantics but is certainly much simpler.
  * Make this computation its own function (int_ceil or similar).
  * Something like ```int(math.ceil(max_concurrent_launches / float(max_launch_batch))```. (You can do this nicely in Python 3 with just ```math.ceil(max_concurrent_launches / max_launch_batch```, but this is not backward compatible to Python 2).